elastic/kibana
# [Console] Added autocompletion for update by query

Fix https://github.com/elastic/kibana/issues/67498

## Summary

Added autocomplete suggestions for update by query.

### How to test

1. Start Kibana
2. Go to Console and type:
 ```
POST <index>/update_by_query
{
 q<ctrl+space>
}
```

You should see completion for query. Inside of the the expanded braces you should see the regular query completion suggestions.
